### PR TITLE
fix navbar styles on android

### PR DESCRIPTION
**Pull request by chirag04**
@brentvatne no need to add fontFamily. System default is Roboto.

### CODE REVIEW

**Comment by satya164** [comment]
An advantage of not specifying font family is, if someone has a custom font set on native side, it'll just work. Advanced use case, but still :)

File: src/ExNavigationBar.js [hunk 1]
```diff
@@ -59,10 +59,18 @@ const titleStyles = StyleSheet.create({
 
   titleText: {
     flex: 1,
-    fontSize: 18,
-    fontWeight: '500',
     color: 'rgba(0, 0, 0, .9)',
-    textAlign: Platform.OS === 'ios' ? 'center' : 'left',
+    ...Platform.select({
+      ios: {
+        fontSize: 18,
```
Comment (chirag04) [inline src/ExNavigationBar.js]: OCD: this should be 17!

**Comment by chirag04** [comment]
@satya164 I took a closer look at it on a device. Appbar default font is `sans-serif-medium`. RN default to `sans-serif`. I think we should set the default fontFamily. 

We already have a way to override the title style. what do you think?

**Comment by satya164** [comment]
@chirag04 True.

> We already have a way to override the title style. what do you think?

I was talking about setting it from native, where you override `sans-serif`, `serif` etc. to be a custom font, so it's applied across everything. I think since you did `sans-serif-medium` in PR, it might work with that approach as well.

**Pull request closed without merging**